projects/inspector-git.manifest.yml

```diff
@@ -0,0 +1,40 @@
+slug: inspector-git
+name: Inspector Git
+description: Inspector Git Description
+
+license: Apache-2.0
+
+repositories:
+  - https://github.com/dxworks/inspector-git-scriptbee-plugin
```

Comment:
This is the original repo: https://github.com/dxworks/inspector-git so it needs to go first